MAINTAINERS

```diff
@@ -61,6 +61,22 @@ MAINTAINERS file. The candidate becomes a maintainer once the pull request is
 merged.
 """
 
+	[Rules.adding-sub-projects]
+
+	title = "How are sub projects added?"
+
+	text = """
+Similar to adding maintainers, new projects can be added to containerd GitHub
+organization. After a project has been announced on the maintainers mailing list,
```

Comment:
I'd be fine for github, only weird thing with github is that issue comments aren't immutable but that's just a small issue, w/e works for the team is fine with me